change `block_number` to hex string

nicodechal opened the pull request:
In Smallbank benchmark, change `block_number` to hex string like YCSB benchmark to avoid request error:
https://github.com/nicodechal/blockbench/blob/ae833572c16d60f76fd3b249f93a4517336e19ee/src/macro/kvstore/core/evm_utils.cc#L289-L291

and then I found in `poll_txs_by_block_number`, uncle in uncles already has quotes, But `GET_BLOCK_BY_HASH_PREFIX` gives uncle one more pair of quotes, so I remove the quotes of uncle to make it run correct. If this is true, then YCSB benchmark also need to be modified.

ug93tad commented:
Thanks for the PR. 

+ Ethereum eth_getBlockByNumber take a decimal representation as well as hex, so the current implementation still works. 

+ The quote was already removed in the StatusThread in smallbank.cc file.

nicodechal commented:
@ug93tad 
- I tested with go-ethereum v1.8.16 and I got only zero results like I mentioned in https://github.com/ooibc88/blockbench/issues/51#issue-381485875 with decimal blocknumber use Smallbank workload.
The Error I got like this:
```
{
  "jsonrpc":"2.0",
  "id":1,
  "error": { 
    "code": -32602,
    "message": "invalid argument 0: hex string without 0x prefix"
  }
}
```
- and the quote I removed is different with the quote in `StatusThread`. The hash I removed quote is uncle's and it's used to find uncle block with `poll_txs_by_block_hash`, while removing quote in `StatusThread` is for counting.
